packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_IPOUHarmonicCoarseOperator_def.hpp

```diff
@@ -282,6 +292,18 @@ namespace FROSch {
                 }
             }
 
+            PartitionOfUnity_->assembledPartitionOfUnityMaps();
+
+           this->kRowMap_ = PartitionOfUnity_->getAssembledPartitionOfUnityMap();
+
+           if (this->ParameterList_->get("Use RepMap",false)) {
+                    Teuchos::RCP<DDInterface<SC,LO,GO,NO> > theInterface =Teuchos::rcp_const_cast<DDInterface<SC,LO,GO,NO> >(interfacePartitionOfUnity->getDDInterface());
+                    this->buildGlobalGraph(theInterface);
+
+                    this->numEnt = interfacePartitionOfUnity->getDDInterface()->getNumEnt();
+                  //  }
+                }
+
```

Comment:
What is the meaning of the Parameter `"Use RepMap"`?

Reply:
This part of the code only needs to be performed if the distribution type is _ZoltanDual_, which includes the build of the repeated map on the coarse level. I changed this to:
` if (!this->DistributionList_->get("Type","linear").compare("ZoltanDual"))`